base/sparse/linalg.jl

```diff
@@ -499,7 +499,196 @@ function norm(A::SparseMatrixCSC,p::Real=2)
     throw(ArgumentError("invalid p-norm p=$p. Valid: 1, Inf"))
 end
 
-# TODO
+# TODO rank
+
+# cond
+function cond(A::SparseMatrixCSC, p::Real=1)
```

Comment:
What about using `2` as the default here. Eventually, we should support the 2 norm and until we do, people won't accidentally get different norm than they expect.